ansible-tower/policies/AT-Awareness_and_Training/component.yaml

```diff
@@ -8,96 +8,124 @@
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+    - key: a
+      text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
```

Comment:
@shawndwells should we say `${product_name}` instead of `component-level`?

Reply:
Yes, probably.

The current language is from a template. 

What about using the template for now, and we can ``s/component-level/${product}/g`` once there are control responses?

Reply:
@shawndwells sure. why not.